libhb/hb.c

```diff
@@ -28,6 +28,11 @@
 #endif
 #endif
 
+#if HB_PROJECT_FEATURE_VAAPI
+static const char * vaapi_device0 = NULL; // "/dev/dri/renderD128";
+AVBufferRef *vaapi_device_ctx0 = NULL; // reused in vaapi_common.c
```

Comment:
Please don't add any global variable. It should be possible to run multiple encodes in a single process.